vendor/vendor.json

```diff
@@ -7,6 +7,12 @@
 			"revision": ""
 		},
 		{
+			"checksumSHA1": "Vrt1uoeOOk9fp1aa9d7huQhzexU=",
+			"path": "code.gitea.io/sdk/gitea",
```

Comment:
should this be `github.com/go-gitea/go-sdk`? 

when I visit `code.gitea.io/sdk/gitea` I don't see any source code (maybe I'm not understanding the UI though). I would not feel comfortable with a dependency that developers cannot trace back to the origin source.